infer_subc/utils/stats_helpers.py

```diff
@@ -421,23 +424,19 @@ def batch_summary_stats(csv_path_list: List[str],
     contact_tabs = []
     dist_tabs = []
     region_tabs = []
-    
```

Comment:
Re add the following lines and remove them from below (lines 436-439)

Reply:
Update has been made!